File: JavaScript/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var a = parseInt(lines.shift());
var b = parseInt(lines.shift());
var c = parseFloat(lines.shift());
var x = b*c

console.log("NUMBER = " + a)
console.log("SALARY = U$ " + x.toFixed(2))

File: C/1008.c
#include <stdio.h>
 
int main() { 
    int a, b;
    double c, x;
    
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%lf", &c);

    x = b*c;

    printf("NUMBER = %d\n", a);
    printf("SALARY = U$ %.2lf\n", x);

    return 0;
}